dbt/README.md

```diff
@@ -206,12 +206,10 @@ dbt build --select default.* --resource-types model seed
 > [!NOTE]
 > If you are building a [Python model](https://docs.getdbt.com/docs/build/python-models),
 > your model may require external dependencies be available on S3.
-> To make these dependencies available to your model, run
-> `../.github/scripts/deploy_dbt_model_dependencies.sh <your_model_name>`
-> in the context of the `dbt/` directory before you run `dbt build`.
-> You can also run the script with no arguments to make dependencies available
-> for all Python models in the project, but this will take a longer time to
-> complete and will use extra storage space on S3.
+> To make these dependencies available to your model, run the
+> `build-and-test-dbt` workflow on your branch to deploy any Python dependencies
+> that you've added to [the `config.packages` attribute](https://docs.getdbt.com/docs/build/python-models#configuring-packages)
+> on your model.
```

Comment:
Juniors don't have permissions to modify the `ccao-athena-dependencies-us-east-1` bucket, so running the workflow will be the easiest way to update dependencies. I wonder if it would be better to just give them permissions to modify the bucket, given that it's easy to recreate its contents?

Reply:
Cool! I'll leave it as-is for now then, but we can keep an eye on it and adjust their permissions if it proves to be easier for them to run the script directly.